UnitTest/main.cpp

```diff
@@ -69,6 +69,7 @@ TEST_F( OroTestBase, kernelExec )
 	int blockCount;
 	OROCHECK( oroOccupancyMaxActiveBlocksPerMultiprocessor( &blockCount, kernel, 128, 0 ) );
 	printf( "%d blocks per multiprocessor\n", blockCount );
+	OROASSERT( 0 < blockCount );
```

Comment:
```suggestion
	int blockCount {};
	OROCHECK( oroOccupancyMaxActiveBlocksPerMultiprocessor( &blockCount, kernel, 128, 0 ) );
	printf( "%d blocks per multiprocessor\n", blockCount );
	OROASSERT( 0 < blockCount );
```
Thanks for adding the test. It looks good.
One suggestion: It might be better to initialize `blockCount ` to zero and then check if that value is greater than zero after the call.